README.md

```diff
@@ -1,43 +1,56 @@
+### Previous Plugins
+Please note [Previous plugins](https://github.com/sa2c/CellProfiler-RunOnCluster/archive/master.zip) do not fully work with the precompiled CellProfile executable (version 3.1.8), which has default settings for the SCW Sunbird cluster, can be found in [releases](https://github.com/sa2c/CellProfiler-RunOnCluster/releases/download/v1.0/CellProfiler.exe). These executables require that a Java Runtime Environment is installed. You can get one for example form [Java](https://www.java.com/en/download/).
 
-Follow the instructions for installing CellProfiler on the [wiki](https://github.com/CellProfiler/CellProfiler/wiki). Choose you operating system on the right side panel. Once you have installed CellProfiler, set the plugins directory in Cellprofiler preferences. Download [the plugins](https://github.com/sa2c/CellProfiler-RunOnCluster/archive/master.zip) and copy move the files to your plugins directory.
+### Latest Plugins
+Please download the [plugins](https://codeload.github.com/sa2c/CellProfiler-RunOnCluster/zip/refs/heads/cellprofiler-4.2.2rc) to your plugins directory. In the plugins directory, install the requirements for the plugins:
 
-In the plugins directory, install the additional requirements for the plugins:
 ```
-python2 -m pip install -r requirements.txt
+python -m pip install -r requirements.txt
 ```
 
-## Usage
+Please be aware that if `pip install` does not install the `Rynner` or the `libsubmit` library properly (i.e. there is no ``Rynner`` folder inside ``site-packages``.). You should copy the `Rynner` folder in this repository and paste it inside ``side-packages``. You may also need to copy the `libsubmit` folder in this branch to ``side-packages``.
 
-### Submitting to Cluster
+## Usage
+### Submitting josb to cluster
 
 Once you have tested your pipeline on your local machine, add all images to be processed into the Images plugin in the usual way. Add the `RunOnCluster` module in the `Other` category to the end of the pipeline.
 
 The module has four settings:
- * Run Name: An identifier that allows you to recognize the pipeline and image batch.
+ * Run Name: An identifier that allows you to recognise the pipeline and image batch.
  * Number of images per measurement: If several image files are required for a single measurement, adjust this to the number of images required.
  * Image type first: Select `Yes` if the image type appears before the measurement number in the image file name. Select `No` if the measurement number appears before the image type.
  * Maximum Runtime (hours): The amount of time to reserve a node for on the cluster. The actual runtime can be lower, but not larger than this. If the run takes longer than the time given, it will be terminated before completion. Must be less than 72.
+ * Project Code: Specify your Supercomputing Wales project code, for example, scw1234.
```

Comment:
Trivial comment, but we should not use an actual valid SCW project code here. `scw0123` is not a valid project, but still illustrates the principle just as well.